src/urg_node_driver.cpp

```diff
@@ -390,7 +401,7 @@ void UrgNode::updateDiagnostics()
 {
   while (!close_diagnostics_)
   {
-    diagnostic_updater_->update();
+    diagnostic_updater_.update();
```

Comment:
force_update()? I am getting compilation errors saying update() is private function

Reply:
Yup. See here: https://github.com/ros/diagnostics/pull/114/files#r320031172